tools/fpgad/plugins/fpgad-vc/fpgad-vc.c

```diff
@@ -432,7 +432,7 @@ STATIC fpga_result vc_enum_sensors(vc_device *vc)
 	snprintf(glob_pattern, sizeof(glob_pattern),
 		 "/sys/bus/pci/devices/%s/fpga_region/region*/dfl-fme.*/"
```

Comment:
Latest driver changes dfl-fme  to dfl_dev .0, please change it  "dfl*

Reply:
It's still dfl-fme.0 at 5.10-rc1.  Are you sure it's changing?